CHANGELOG.md

```diff
@@ -182,4 +184,28 @@ the package that it is today.
 
 [Unreleased]: https://github.com/ericmjl/pyjanitor/compare/v0.20.11...HEAD
 
-[0.20.11]: https://github.com/ericmjl/pyjanitor/compare/v0.20.10...v0.20.11
+[v0.20.11]: https://github.com/ericmjl/pyjanitor/compare/v0.20.10...v0.20.11
+
+[v0.20.10]: https://github.com/ericmjl/pyjanitor/compare/v0.20.9...v0.20.10
+
+[v0.20.9]: https://github.com/ericmjl/pyjanitor/compare/v0.20.8...v0.20.9
+
+[v0.20.8]: https://github.com/ericmjl/pyjanitor/compare/v0.20.7...v0.20.8
+
+[v0.20.7]: https://github.com/ericmjl/pyjanitor/compare/v0.20.5...v0.20.7
+
+[v0.20.6]: https://github.com/ericmjl/pyjanitor/compare/v0.20.5...v0.20.7
```

Comment:
Note: There is no `v0.20.6` tag. Instead of having the link tied to an unhelpful URL, I elected to route `v0.20.6` in the CHANGELOG to compare `v0.20.5` to `v0.20.7`. This means the link for `v0.20.6` and `v0.20.7` in the CHANGELOG will be the same